src/storm/adapters/RationalNumberForward.h

```diff
@@ -30,8 +30,38 @@ typedef cln::cl_I ClnIntegerNumber;
 typedef __gmp_expr<mpq_t, mpq_t> GmpRationalNumber;
 typedef __gmp_expr<mpz_t, mpz_t> GmpIntegerNumber;
 #endif
+
+/*!
+ * Interval type
+ */
 typedef carl::Interval<double> Interval;
 
+namespace detail {
```

Comment:
Do we maybe want to put this into an own interval-header? 

Advantages:

- Probably, no one will look in rationalnumber for interval based stuff
- The places that need intervals and need rationals need not coincide

Downsides:
- More headers
- With exact intervals on the way, the distinction is less clear.

Reply:
If we keep it here, maybe we should call it NumberRepresentationForward.h ? (and modify the other headers as well?)